apps/meteor/client/sidebar/footer/voip/index.tsx

```diff
@@ -62,10 +62,13 @@ export const VoipFooter = (): ReactElement | null => {
 	};
 
 	const tooltips = {
-		mute: t('Mute'),
-		holdCall: t('Hold_Call'),
-		holdCallEEOnly: t('Hold_Call_EE_only'),
-		acceptCall: t('Accept_Call'),
+		mute: t('Turn_off_microphone'),
+		unmute: t('Turn_on_microphone'),
+		resume: t('Resume'),
+		hold: t('Hold'),
+		holdEEOnly: t('Hold_EE_only'),
+		accept: t('Accept'),
+		decline: t('Decline'),
 		endCall: t('End_Call'),
 	};
 
```

Comment:
could you remove this object and use t(...key) inside the component? that 'configuration' was done by mistake and I'm afraid its getting worse